README.md

```diff
@@ -2,9 +2,12 @@
 
 ## What is a Pairing Project?
 
-In software development, the word 'pair' is short for 'pair programming', and is used to generically mean working together. The technique of pairing can vary, but there are typically 2 distinct roles. The first developer is called a *driver*. They will physically write all the code for the pair. The second person is called a *navigator*, but may have the role of an observer, a navigator, or a researcher. This person's responsibility is to guide strategy, coming up with ideas for improvements and looking out for any issues that may arise in the code's design.
+[Pair programming](http://wiki.c2.com/?PairProgramming) is an [Extreme Programming Practice](http://wiki.c2.com/?ExtremeProgrammingPractice). Two developers actively collaborate on the same development effort. Traditionally pairing takes place at the same workstation with the pair sitting together. Developers can pair on individual computers when screen sharing used. Pairing can even be remote. What matters is the pair is seeing the same screen and communicating as they work.
```

Comment:
Nit:

```
Developers can pair on individual computers when screen sharing *IS* used.
```
And in the final sentence, what do you think about swapping the order:

```
What matters is the pair is communicating as they work and viewing the same screen.
```